server/services/datacommons.py

```diff
@@ -531,7 +531,7 @@ def get_statvar_group(stat_var_group, places):
     url = API_ROOT + API_ENDPOINTS['get_statvar_group']
     req_json = {
         'stat_var_group': stat_var_group,
-        'places': places,
+        'entities': places,
```

Comment:
change the argument "places" to "entities" as well?

Reply:
done